doc/mgr/dashboard.rst

```diff
@@ -319,6 +319,59 @@ You need to tell the dashboard on which url Grafana instance is running/deployed
 The format of url is : `<protocol>:<IP-address>:<port>`
 You can directly access Grafana Instance as well to monitor your cluster.
 
+Enabling Single Sign-On (SSO)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. note::
+  Ceph Dashboard SSO support relies on onelogin's
+  `python-saml <https://pypi.org/project/python-saml/>`_ library.
+  Please ensure that this library is installed on your system, either by using
+  packages provided by your operating system or using Python's ``pip``
+  installation tool:
+
+  **Python 2:**
+
+  ``$ pip2 install python-saml``
+
+  **Python 3:**
+
+  ``$ pip3 install python3-saml``
+
+Ceph Dashboard supports the `SAML 2.0 <https://en.wikipedia.org/wiki/SAML_2.0>`_
+protocol to integrate with an existing Identity Provider (IdP).
```

Comment:
Reading this paragraph again, I wonder if it would make sense to emphasize that it's still required to create the user accounts locally and assign the appropriate roles to them? We should describe that the current SSO implementation is only used for **authentication**, while the **authorization** is still performed by the dashboard.

Reply:
Also, I think this paragraph should be moved on top of this section, before the note about the `python-saml` dependency. How about extending it as follows?
```
The Ceph Manager Dashboard supports external authentication of users via the
`SAML 2.0 <https://en.wikipedia.org/wiki/SAML_2.0>`_ protocol. You need to create
the user accounts and associate them with the desired roles first, as authorization
is still performed by the Dashboard. However, the authentication process can be
performed by an existing Identity Provider (IdP).
```

Reply:
@LenzGr I've applied your documentation improvement suggestion, much better now. Thanks.